.gitlab-ci.yml

```diff
@@ -369,36 +366,36 @@ test:helm_chart_upgrade_from3.4:
   script:
     - tests/ci-make-deps.sh
     - |
-      echo "INFO - installing mender 3.4"
+      echo "INFO - installing mender tag ${MENDER_CURRENT_LTS_RELEASE_TAG:-mender-3.6.0}"
```

Comment:
```suggestion
      echo "INFO - installing mender tag ${MENDER_CURRENT_LTS_RELEASE_TAG:-mender}-${TARGET_VERSION:-3.6.0}"
```

Reply:
I'll leave this refactoring to @oldgiova